src/neuroconv/tools/nwb_helpers.py

```diff
@@ -47,8 +47,11 @@ def get_default_nwbfile_metadata() -> DeepDict:
 
 def make_nwbfile_from_metadata(metadata: dict) -> NWBFile:
     """Make NWBFile from available metadata."""
-    metadata = dict_deep_update(get_default_nwbfile_metadata(), metadata)
-    nwbfile_kwargs = metadata["NWBFile"]
+    nwbfile_kwargs = metadata.get("NWBFile") or dict()
```

Comment:
```suggestion
    assert "NWBFile" in metadata and "session_start_time" in metadata["NWBFile"], "The metadata dictionary must contain, at a minimum, a 'session_start_time' in metadata['NWBFile']."
    nwbfile_kwargs = metadata["NWBFile"]
```

Reply:
OR do the ultimate best thing, which would be full metadata JSON validation of either a (i) passed schema, or (ii) just the  `NWBFile` subset of the schema